client/react/frontpage/components/PromoBanner.js

```diff
@@ -15,8 +15,8 @@ import './PromoBanner.scss';
 //  }
 const bannerData = [
   {
-    img: '/img/promo/web-wine-n-blanket.png',
-    link: 'https://www.facebook.com/events/2067576166837819/',
+    img: '/img/promo/sotm/sotm_may_2018.jpg',
+    link: 'https://uclaradio.com/shows/489',
```

Comment:
I'd relative link this: `/shows/489` instead of `https://uclaradio.com/shows/489`.

Reply:
other than that, though, lgtm!